Resources/Prototypes/Maps/Pools/default.yml

```
@@ -5,19 +5,19 @@
   - Core
   - Fland
   - Marathon
-  - Meta
-  - Oasis
-  - Omega
-  - Origin
-  - Saltern
-  - Packed
-  - Reach
-#  - Train # Goobstation - all my friends hate train
+  #- Meta TODO: pirasmas meta
```

yamllint (1.35.1)

[warning] missing starting space in comment

(comments)